demos/createSolution/index.html

```diff
@@ -43,6 +43,8 @@ <h1>Convert to Solution</h1>
           <input class="form-control" type="text" id="itemOrGroupId">
           <label for="id">Or pick from the list below:</label> 
           <select name="Solution Item" id="id" class="form-control">
+            <option value="49de38329ee14b39b7ec2f122c5ee819">Site - Winter Fun  ( 49de38329ee14b39b7ec2f122c5ee819) </option>
```

Comment:
Are all of the dependencies in these examples public? I can't create any of them